### DISCUSSION TITLE
collections only in collection ????

### DISCUSSION THREAD

**Discussion by cracyfloyd** [meisnate12/Plex-Meta-Manager#502] [Q&A]
It is possible that all collections are generated only visible in collection and not in movies???

**Comment by meisnate12** [comment]
if the collection visibility is set to hide for all of them

**Reply by russellmoran99** [reply]
I am having a similar issue. All of the smart collections are in the Library folder as well. I have Visible On with nothing checked.  I have a "Kids Movies" library in Plex. My 7 year old logs into his local account on my LAN and goes to Library and has to scroll down about 7 pages until he finds the beginning of the true library. This should only be in Collections in my opinion. How can we change this? Thanks so much.

**Comment by cracyfloyd** [comment]
Yes i nthink. i see it when my sorting is title. Then i see the collections in movies and collection. when i use an another sorting then its only to see in collection.

**Reply by meisnate12** [reply]
Yes sorting by title will show the collections but other sort modes will not 

You have to go to the advance options when you edit the collection and change collection mode to hide

**Reply by russellmoran99** [reply]
I went to Kids Movies --> Library --> Edit --> Advanced --> Changed from Library default to Hide Collection.  It worked for the Kids Movies library.  Now when I try to repeat this step and go to "Moves", "4K Movies", etc...the Advanced already shows "Hide Collection".  It is not hidden in those Library's though.

**Reply by meisnate12** [reply]
So what you changed was the default library setting you can also set that setting per collection. By default it should be set to library default but if that's been changed then you would need to change it back

**Reply by russellmoran99** [reply]
I will go back and change it.  How do I change it per collection and per library?  In other words, I may want Movies and Kids Movies to show different collections in the Library section.

**Reply by meisnate12** [reply]
Edit the collection and go to advance it's in there

**Reply by russellmoran99** [reply]
If I go to Movies --> Collections --> Oscars --> Edit --> Advanced --> change Collection mode to "Hide items in this collection".   It still appears in my Movies --> Library.  I can then go to Kids Moves --> Collections --> Oscars --> Edit --> Advanced --> and it is set as Hide items in this collection.   I did not change it in Kids Movies, it was originally Library Default.  I only changed it in Movies.  Never the less, it appears in all my library sections.  I'm not sure how to get this fixed.  I sincerely appreciate your help.

**Reply by meisnate12** [reply]
No not hide items in this collection you need to select the hide collection option

**Reply by russellmoran99** [reply]
I may not be doing it correctly then.  If I go to Movies and chance the collection advanced to Hide collection, then go to Kids Movies, it has taken the same input of "Hide collection."  When I change one collection, it changes ALL collections.  I'm not sure why.  Can you be exact on how you change the individual collections per library?  I may be missing a simple option.  Thanks

**Reply by russellmoran99** [reply]
Here's an idea, may or may not have any relevance.  I created separate yml files for each library and the config file points to each .yml file.  For example I have Movies.yml, 4K Movies.yml, Kids Movies.yml, 4K Kids Movies.yml.  I copied and pasted the original Movies.yml content to each file.  They are all identical.  Do I need to change something in the files?  I'm wondering if they're all linking the collections somehow?

**Comment by russellmoran99** [comment]
I believe I have found the problem/solution/fix.  I have to go to each individual library and it the collection is set to Hide, I have to change it back to library default.  Save that and then open again and change back to hide.  This has to be done to each individually.  This is now working.  It might be a Plex bug?

**Reply by meisnate12** [reply]
this sounds like some kind of bug
